aiounifi/dpi.py

```diff
@@ -0,0 +1,76 @@
+"""DPI Restrictions as part of a UniFi network."""
+from typing import List
+
+from .api import APIItem, APIItems
+
+GROUP_URL = "/rest/dpigroup"  # List DPI Group configuration
+APP_URL = "/rest/dpiapp"  # List DPI App configuration
+
+
+class DPIRestrictionGroup(APIItem):
+    """Represents a DPI App configuration."""
+
+    @property
+    def id(self) -> str:
+        return self.raw["_id"]
+
+    @property
+    def attr_no_delete(self) -> bool:
+        return self.raw.get("attr_no_delete", False)
+
+    @property
+    def attr_hidden_id(self) -> str:
+        return self.raw.get("attr_hidden_id", "")
+
+    @property
+    def name(self) -> str:
+        return self.raw["name"]
+
+    @property
+    def site_id(self) -> str:
+        return self.raw["site_id"]
+
+    @property
+    def dpiapp_ids(self) -> List[str]:
+        return self.raw.get("dpiapp_ids", [])
+
+    @property
+    def enabled(self) -> bool:
+        return self.raw["enabled"]
+
+
+class DPIRestrictionGroups(APIItems):
+    """Represents DPI Group configurations."""
+
+    KEY = "name"
```

Comment:
Can a user rename the app at any time? Then maybe the _id should be used?

Reply:
Fair enough. This was taken from WLANs.

Reply:
Good point, though WLANs is not in use :)